cmd/gateway/main.go

```diff
@@ -55,10 +65,10 @@ func main() {
 	apiSpecServer := api.NewApiSpecServer(apiMux, OPENAPI_SPEC_FILE)
 	apiSpecServer.Routes()
 
-	mgmtServer := api.NewManagementServer(cm, apiMux, wsConfig.ServiceToServiceCredentials)
+	mgmtServer := api.NewManagementServer(cm, apiMux, cfg.ServiceToServiceCredentials)
 	mgmtServer.Routes()
 
-	jr := api.NewJobReceiver(cm, apiMux, kw, wsConfig.ServiceToServiceCredentials)
+	jr := api.NewJobReceiver(cm, apiMux, kw, cfg.ServiceToServiceCredentials)
```

Comment:
What do you think about passing the entire config object to the ManagementServer and JobReceiver constructors?